docs/How-To-Create-Evergreen-Projects.md

```diff
@@ -0,0 +1,22 @@
+# How to Create Evergreen Projects
+
+This guide is meant to help users create their own projects on Evergreen without creating a ticket and waiting. 
```

Comment:
can we actually drop "without creating a ticket and waiting" bc one of our steps is still that